### PR TITLE
Fix policies to work with the container-selinux > 2.189.0

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/k3s-io/k3s-selinux/issues/36

1- Add policy dir for slemicro with requirement of container-selinux < 2.191
2- Add policy dir for coreos with requirement of container-selinux >= 2.191
3- Fix microos policy to remove the conflict and add  requirement of container-selinux >= 2.191
4- Add requirement for container-selinux < 2.191 for centos8 policy

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Review by dereknola — APPROVED** [review]

File: policy/centos8/k3s-selinux.spec [hunk 1]
```diff
@@ -31,14 +32,14 @@ Source0:	k3s.pp
...
-BuildRequires: container-selinux >= %{container_policy_epoch}:%{container_policyver}
+BuildRequires: container-selinux >= %{container_policy_epoch}:%{container_policyver}, container-selinux < %{container_policy_epoch}:%{container_policyver_max}
 BuildRequires: git
 BuildRequires: selinux-policy >= %{selinux_policyver}
 BuildRequires: selinux-policy-devel >= %{selinux_policyver}
 
 Requires: policycoreutils, libselinux-utils
 Requires(post): selinux-policy-base >= %{selinux_policyver}, policycoreutils
-Requires(post): container-selinux >= %{container_policy_epoch}:%{container_policyver}
+Requires(post): container-selinux >= %{container_policy_epoch}:%{container_policyver}, , container-selinux < %{container_policy_epoch}:%{container_policyver_max}
```
Comment (pjbgf) [inline policy/centos8/k3s-selinux.spec:42]: Maybe a typo?

```suggestion
Requires(post): container-selinux >= %{container_policy_epoch}:%{container_policyver}, container-selinux < %{container_policy_epoch}:%{container_policyver_max}
```

File: policy/coreos/k3s-selinux.spec [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+%define container_policy_epoch 2
+
+Name:   k3s-selinux
+Version:	%{k3s_selinux_version}
+Release:	%{k3s_selinux_release}.el8
+Summary:	SELinux policy module for k3s
+
+Group:	System Environment/Base
+License:	ASL 2.0
+URL:		http://k3s.io
```
Comment (pjbgf) [inline policy/coreos/k3s-selinux.spec:29]:
```suggestion
URL:		https://k3s.io
```

File: policy/slemicro/k3s-selinux.spec [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+%define container_policyver_max 2.191.0-1
+
+Name:   k3s-selinux
+Version:	%{k3s_selinux_version}
+Release:	%{k3s_selinux_release}.sle
+Summary:	SELinux policy module for k3s
+
+Group:	System Environment/Base
+License:	ASL 2.0
+URL:		http://k3s.io
```
Comment (pjbgf) [inline policy/slemicro/k3s-selinux.spec:29]:
```suggestion
URL:		https://k3s.io
```

**Review by pjbgf — COMMENTED** [review]
A few small nits:

File: policy/centos8/k3s-selinux.spec (hunk 1 above)
Comment (galal-hussein) [reply policy/centos8/k3s-selinux.spec:42]: thanks for reviewing, will fix

**Review by pjbgf — APPROVED** [review]

**Comment by wranders** [comment]
Is there anything internal that's blocking this?

**Pull request merged**